internal/command/statics.go

```diff
@@ -1,12 +1,31 @@
 package command
 
 import (
+	"bytes"
 	"context"
 	"io"
+	"strings"
 
 	"github.com/caos/zitadel/internal/domain"
+	"github.com/go-oss/image/imageutil"
 )
 
+func (c *Commands) RemoveExif(file io.Reader, size int64, contentType string) (io.Reader, int64, error) {
```

Comment:
this function does not create events or anything so IMO it should either be a private function which is called by `UploadAsset` to always ensure exif data is removed or if only relevant for upload handler we could create it there